agent/wptdriver/wpt_driver_core.cc

```diff
@@ -819,6 +838,10 @@ void WptDriverCore::PreTest() {
       CloseHandle(process);
   }
 
+  // Minimise all windows on desktop
+  HWND hwnd = FindWindow(_T("Shell_TrayWnd"), NULL);
+  LRESULT res = SendMessage(hwnd, WM_COMMAND, (WPARAM)419, 0);

```

Comment:
This is kind of hacky and not supported (though seems to be used frequently on stack overflow).  I'd feel better if this used GetDesktopWindow(), got the first child, enumerated all of the next windows and minimized each one like the dialog closing code: https://github.com/WPO-Foundation/webpagetest/blob/master/agent/wptdriver/wpt_driver_core.cc#L698